website/config.toml

```diff
@@ -1,12 +1,24 @@
 baseURL = "https://staticcheck.io"
 languageCode = 'en-us'
 title = 'Staticcheck'
-theme = 'docsy'
```

Comment:
Does the theme get used automagically, just because we're importing the module?

Reply:
No. This section, which was added at the end of `config.toml` imports the modules.

```
[module]
[[module.imports]]
path = "github.com/google/docsy"
[[module.imports]]
path = "github.com/google/docsy/dependencies"
```

**Note**: you could also alter the theme line to: 

```
theme = "github.com/google/docsy"
```

Both are valid (and more or less equivalent) solutions.